src/components/BAccordion/BAccordion.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <section
+    :id="id"
+    class="acc-flex-column"
+  >
+    <div
+      role="heading"
+      class="acc-flex-center"
+    >
+      <button
+        :id="`${_uid}-heading`"
+        class="acc-header"
+        :aria-expanded="expanded"
+        :aria-controls="`${_uid}-content`"
+        aria-disabled="false"
```

Comment:
If this is always collapsible, we don't need `aria-disabled` at all

Reply:
ok